.spectral.yaml

```
@@ -31,7 +31,7 @@ rules:
 
   ## Allowed and required verbs
   collection-verbs-required:
-    description: Resource collections must have POST, GET, DELETE verbs.
+    description: Resource collections must have POST and GET verbs. # Updated description
```

Don't comment about what changed
```suggestion
    description: Resource collections must have POST and GET verbs.
```